content/en/docs/tasks/observability/distributed-tracing/configurability/telemetry-api/index.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Using the Telemetry API [Experimental]
+description: How to configure tracing options using the Telemetry API (experimental).
+weight: 10
+keywords: [telemetry,tracing]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
+---
+
+Istio provides a Telemetry API that enables flexible configuration of tracing behavior. The experimental
+Telemetry API offers controls over tracing options such as sampling rates and custom tags for individual
+spans, as well as backend provider selection. The options are configurable down to the the individual
+workload level and provide override behavior.
+
+## Before you begin
+
+1.  Ensure that your applications propagate tracing headers as described [here](/docs/tasks/observability/distributed-tracing/overview/).
+1.  Follow the tracing installation guide located under [Integrations](/docs/ops/integrations/) based on your preferred tracing backend
+    to install the appropriate tracing provider.
+
+## Telemetry API: Tracing Overview
+
+The Telemetry API offers tracing behavior configuration control over the following at the mesh, namespace, and workload levels:
+
+- **provider selection** - allows selection of backend providers (providers are configured in `MeshConfig`) for reporting.
+
+- **sampling percentage** - allows control the rate of trace sampling applied to received requests *for which no prior sampling decision has been made*.
+
+- **custom tags** - allows control over any custom tags to add to each generated tracing span.
+
+- **tracing participation** - allows opting services out of reporting spans to the selected tracing providers.
+
+### Workload Selection
+
+Individual workloads within a namespace are selected via a [`selector`](/docs/reference/config/type/workload-selector/#WorkloadSelector) which allows selection of workloads based on labels.
+
+It is not valid to have two different `Telemetry` resources select the same workload using `selector`. Likewise, it is not valid to have two
+distinct `Telemetry` resources in a namespace with no `selector` specified.
```

Comment:
What's the behavior when this happens? Can we elaborate on that too.

Reply:
It is undefined. I'm worried that documenting the behavior could lead to a dependency on the behavior itself. As part of alpha, I'd like use to consider rejecting such configuration at validation. Thoughts?

Reply:
Yes, rejecting is preferred.